### DISCUSSION TITLE
I can't log in after changing my email address

### DISCUSSION THREAD

**Discussion by FloydKulis** [firefly-iii/firefly-iii#9224] [Firefly III]
I did something very stupid today. I can't log in because I changed my email address. I am informed that I need to confirm my e-mail address and I cannot do it (I do not have an e-mail client set on this domain).
![image](https://github.com/user-attachments/assets/41b3cedb-45fd-45f7-975b-65b000b46e49)
Can this be changed somehow from the database side?

//EDIT
I found it:
![image](https://github.com/user-attachments/assets/f23848d9-5eba-4f6a-96b5-ad5925ae44de)

**Comment by JC5** [comment]
Hi, great question! 

Somebody asked this three days ago. You should be able to find the answer quickly. It's also in the documentation but I haven't got the link ready for you.
